lib/chitter.rb

```diff
@@ -1,6 +1,24 @@
 require 'pg'
 
 class Chitter
+    def self.all
+        if ENV['ENVIRONMENT'] == 'test'
+          connection = PG.connect(dbname: 'chitter_test')
+        else
+          connection = PG.connect(dbname: 'chitter')
+        end
+        result = connection.exec('SELECT * FROM peeps')
```

Comment:
`FROM` looks for a table called `peeps`, but peeps is a column, according to your README.